apps/shabbat/shabbat.star

```diff
@@ -1,8 +1,8 @@
 """
-Applet: Shabbat
+Applet: Shabbat Times 
```

Comment:
You will also want to update `shabbot.go` if you want these to update in the mobile app